projects/pyqt.cmake

```diff
@@ -11,21 +11,36 @@ if(BUILD_OS_WINDOWS)
         "CFLAGS+=${CMAKE_C_FLAGS}"
         "CXXFLAGS+=${CMAKE_CXX_FLAGS}"
     )
+# HAS THIS BEEN REMOVED BEFORE?
```

Comment:
I intentionally removed the BUILD_OS_LINUX section since there is no difference anymore with the OSX part of this.